src/js/providers/html5.js

```diff
@@ -502,14 +526,19 @@ define([
                 });
             }
 
-            if (_canSeek) {
+            if (_getSeekableEnd() >= seekPos) {
+                // We can seek if the end seekable range is at or past the seek position
+                _loadInProgress = true;
+            }
+
+            if (_loadInProgress) {
```

Comment:
:sob: `_loadInProgress` does nothing or is it trying to deal with Android duration ranges coming and going?